pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -236,6 +236,8 @@ type Controller struct {
 	// workers that are responsible for tainting nodes.
 	zoneNoExecuteTainter map[string]*scheduler.RateLimitedTimedQueue
 
+	nodesToRetry *sync.Map
```

Comment:
why pointer?
Why not just sync.Map?

Reply:
IMHO it doesn't matter. Pointer removed.